LEAF_Request_Portal/admin/css/mod_groups.css

```diff
@@ -84,4 +84,52 @@ div.groupBlockWhite h3 {
     font-weight: normal;
     font-family: 'PublicSans-Medium';
 
+}
+
+/* Loading Spinner CSS */
+.loadingModal {
+    display: none;
+    position: fixed;
+    height: 100%;
+    width: 100%;
+    top: 0;
+    bottom: 0;
+    background: rgba(96, 96, 96, 0.8);
+}
+
+/* Show Loading Spinner */
+#body.loading .loadingModal {
+    display: block;
+}
+
+/* Loading Image */
+.loadingImage {
+    margin: 15% auto;
+    height: 248px;
+    width: 250px;
+    border-radius: 10%;
```

Comment:
Border radius values should usually be defined as a pixel or em.  If you define it as a percentage, the overall shape of the arc is oval, not circular.  To fix this, you can set the border radius to 999px which will scale with the size of the overall container.  https://stackoverflow.com/questions/29966499/border-radius-in-percentage-and-pixels-px-or-em

Reply:
Right, I was using px first but then I set it up for a circle at 100%, figured 10% would make sense if I just wanted a little curved edges... can change back to 50px which gives me the same affect.